Actions/Deploy/action.yaml

```diff
@@ -5,10 +5,6 @@ inputs:
     description: Shell in which you want to run the action (powershell or pwsh)
     required: false
     default: powershell
-  token:
```

Comment:
Lines 32 and 40 need to be cleaned up as well. Also, run the tests in Tests\Deploy.Action.Test.ps1 locally to verify they pass.

Reply:
@cegekaJG - there is a CI test failing. That test checks whether the .ps1 script matches the .yaml file with parameters etc. once you have removed the token in the lines mentioned by Maria, the test will pass.

Reply:
My mistake, it should work now. The test was successful as well.